homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -175,4 +175,4 @@ def native_value(self) -> Any | None:
 
         raw_value = self.coordinator.data[self.module_id][self.data_id]
 
-        return self._formatter(raw_value) if self._formatter else raw_value
+        return self._formatter(raw_value)
```

Comment:
I struggled with this one!
I think it would benefit from moving to `EntityDescription`: #78837